docs/README.md

```diff
@@ -5,4 +5,35 @@ This directory contains advanced docs around the Functions Framework.
 - TODO: Run Multiple Cloud Functions [#23](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/issues/23)
 - TODO: Pub/Sub Trigger [#37](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/issues/37)
 - TODO: Deploy to Cloud Run [#28](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/pull/28)
-- TODO: Local Debug [#15](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/issues/15)
+
+## Debugging functions
+
+The Functions Framework works with standard tooling that you might use when writing a function for a Node.js environment. You can attach a debugger to your function by following these steps.
+
+1. Write a Node.js function:
```

Comment:
This should be more instructional, like:

1. Create an `index.js` file:

Reply:
Updated.